src/main/kotlin/com/demonwav/mcdev/util/Util.kt

```diff
@@ -35,6 +35,32 @@ fun invokeLater(func: () -> Unit) {
     ApplicationManager.getApplication().invokeLater { func() }
 }
 
+/**
+ * Returns an untyped array for the specified [Collection].
+ */
+internal fun Collection<*>.toArray(): Array<Any> {
```

Comment:
This isn't used in this PR, but I'm assuming it's used in the other two PR's.

I think this is dangerous, though. You're creating an `Array<Any>`, so this array cannot hold nullable values. `java.util.Collection#toArray()` returns (from Kotlin's perspective) `Array<Any!>!` which means it will allow treating it as either nullable or null-safe, since it comes from un-annotated Java code, so there is no way for it to know either way. If we know the contract on the method is null-safe, we can happily treat it as such, but that isn't the case here. That means if this is used and `#toArray()` returns an array with nulls, this may cause failures in Kotlin's nullability checks later down the line, since it's assumed to be non-null only.